addon/index.js

```diff
@@ -18,7 +75,17 @@ moment.prototype.compare = compare;
 moment.compare = compare;
 
 moment.prototype.clone = function clone() {
-  return moment(this);
+  return augmentedMoment(this);
+}
+
+moment.prototype.copy = function copy(deep) {
+  return this.clone();
 }
 
-export default moment;
+const augmentedMoment = createAugmentedMomentFrom(moment);
+
+['utc', 'unix'].forEach((methodName) => {
```

Comment:
Is optionally `tz` needed here since it also returns a moment object?

Reply:
`tz`return a `moment.utc` instance (cf [moment-timezone.js#L488](https://github.com/moment/moment-timezone/blob/develop/moment-timezone.js#L488)) so no need here